### ISSUE TITLE
CoP 8-2 Escort Quest, Quasi's won't repop.

### ISSUE THREAD

**Issue by PoppaPip** [EdenServer/community#2713]
### Checklist

- [X] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [X] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [X] I have provided my client version in the details. (type /ver into your game window)

### Details
Ver. 30181205_0

Quasilumin won't repop if they take off without you. We missed one of the gates and our lil bud left us in the dust. Ran back to the Cermet Alcove and got the message "The alcove has recently been activated...". Waited 30 minutes, got the same message. Ran back to the previous alcove that we had already completed, and got the same message. Both of these sources say that the Quasi can be repopped if he dies or reaches his destination. As a less important side note, our Quasi kept stopping randomly (maybe in anticipation of an unclaimed mob?), and I had to repeatedly talk to them as they would sometimes move 3 feet before taking a break.

https://web.archive.org/web/20060110045430/http://promathia.runtimerepairs.com/Missions/8-2.html

https://ffxiclopedia.fandom.com/wiki/A_Fate_Decided

**Comment by PoppaPip** [comment]
EDIT : Show up 24 hours later and tried again, still won't repop, still get the same message.

**Comment by perkuprightmeow** [comment]
To add to this, they lost their Quasi later in the escort, but the first alcove (for the very short escort) was giving the message above when they returned.

I went ahead and opened the first short escort door on my GM from behind to let them progress, and went to check the alcove again to see if the message would change since the door was open, and the Quasi was just up in the room next to a spinning pot (it was not there before). I activated it, and it moved half way across the hallway then despawned with no trigger. 

Next escort quest started normally.

It seems like Quasi's are not despawning if the escort is failed / they're sat at the finished destination?
